{
  "title": "IDA Pro 7.5 Breakpoint Python Script: 'invalid non-printable character U+FEFF'",
  "link": "https://reverseengineering.stackexchange.com/questions/28055/ida-pro-7-5-breakpoint-python-script-invalid-non-printable-character-ufeff",
  "content": "I'm trying to set a python condition breakpoint, but it throws me a weird error and I don't understand what happens:\n\n\n\nEven if I just return True, it throws the same error when I compile or when press \"OK\" twice.\n\n\n\nThe python console works fine. IDC works fine.\n\nIDA Pro v7.5.200728\nPython 3.9.5\n\nAny idea? Thanks in advance.\n\nSolve:\n\nhttps://hex-rays.com/blog/python-3-9-support-for-ida-7-5/\nDownload the patch and replace the files.\n\n",
  "votes": "1",
  "answers": 1,
  "views": "298",
  "tags": [
    "ida",
    "python"
  ],
  "user": "defaultuser001",
  "time": "Jul 27, 2021 at 20:20",
  "comments": [
    {
      "user": "Robert",
      "text": "If you have solved your problem please write a full answer.",
      "time": null
    }
  ],
  "answers_data": [
    {
      "content": "Solve:\n\nhttps://hex-rays.com/blog/python-3-9-support-for-ida-7-5/ Download the patch and replace the files.\n\n",
      "votes": "-1",
      "user": "defaultuser001",
      "time": "Jul 28, 2021 at 2:10",
      "is_accepted": false,
      "comments": []
    }
  ]
}